report/src/main/webapp/.htaccess

```diff
@@ -15,11 +15,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+
 RewriteEngine On
 # If an existing asset or directory is requested go to it as it is
 RewriteCond %{DOCUMENT_ROOT}%{REQUEST_URI} -f [OR]
 RewriteCond %{DOCUMENT_ROOT}%{REQUEST_URI} -d
 RewriteRule ^ - [L]
 
-# If the requested resource doesn't exist, use index.html
-RewriteRule ^ /report.html
+# If the requested resource doesn't exist, use report.html
+RewriteRule ^$ /report.html
```

Comment:
I think this line should be added right after `RewriteEngine On` so it redirects to report.html whenever the request_uri is empty.
And the previous `RewriteRule ^ /report.html` would stay here. 
Please correct me if I'm wrong.